.github/workflows/go.yaml

```
@@ -44,9 +44,7 @@ jobs:
       uses: actions/setup-go@v6
       with:
         go-version: ${{ matrix.go-versions }}
-    - name: Run go vet
-      run: go vet ./...
     - name: Run yaml-test-suit tests
       run: make test-yts
     - name: Run go test
-      run: GO111MODULE=on go test -v -race .
+      run: GO111MODULE=on go test -vet=off -v -race .
```

The GO111MODULE=on is no longer needed 😅

```suggestion
      run: go test -vet=off -v -race .
```

---

Also, I feel I might have forgot to update this like here when I worked on moving libyaml to internal

It should be this here

```suggestion
      run: go test -vet=off -v -race ./...
```